jina/types/arrays/memmap.py

```diff
@@ -64,12 +64,12 @@ def __init__(self, path: str, key_length: int = 36):
         self._header_path = os.path.join(path, 'header.bin')
         self._body_path = os.path.join(path, 'body.bin')
         self._key_length = key_length
-        self._invalidated_mmap = False
-        self._last_mmap = None
+        self._invalidated_mmap = False  # a boolean flag to say mmap is invalidated, set on the change of mmap
```

Comment:
having only one of these should be enough right? having `self._last_mmap = None` means it was invalidated the last time